src/main/scala/org/broadinstitute/dsde/rawls/model/MethodRepoModel.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.model
+
+import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
+import org.joda.time.DateTime
+
+import scala.annotation.meta.field
+
+object AgoraEntityType extends Enumeration {
+  type EntityType = Value
+  val Task = Value("Task")
+  val Workflow = Value("Workflow")
+  val Configuration = Value("Configuration")
+}
+
+@ApiModel(value = "AgoraEntity")
+case class AgoraEntity(
```

Comment:
I agree that we should use "Method Repo" as much as possible, for the same reason.  In this case, these mirror AgoraEntity/AgoraEntityType in the Agora repo, so I wanted to keep that connection.